aptos-move/move-crates/Cargo.toml

```diff
@@ -0,0 +1,45 @@
+[package]
+name = "move-crates"
```

Comment:
nit: Maybe have a better description here? e.g., `A custom move crate that wraps the external move dependencies?"

Reply:
maybe even a better name.. `move-deps` or even "move-lang"

Reply:
yes, sir